langium/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ### Unreleased
 
+- Leader: Cass Braun
```

Comment:
Need to create a new minor version (e.g. 0.2.1 with the features description included)

This version number should also be bumped in the `package.json`.